packages/contracts/test/Airdrop.test.ts

```diff
@@ -22,18 +22,18 @@ describe('Airdrop', function () {
     let accounts: ethers.Signer[]
 
     let numUsers = 0
-    let attesterAddress, unirepContractCalledByAttester
+    let attesterAddress, unirepContractCalledByAttester: Unirep
     const airdropPosRep = 20
     const epkNonce = 0
     const attestingFee = ethers.utils.parseEther('0.1')
 
     before(async () => {
         accounts = await hardhatEthers.getSigners()
 
+        const _treeDepths = getTreeDepthsForTesting()
         const _settings = {
             attestingFee,
         }
-        const _treeDepths = getTreeDepthsForTesting()
 
         unirepContract = await deployUnirep(
             <ethers.Wallet>accounts[0],
```

Comment:
I see this logic reuse in all files, let create function for it. 
And store it in test/setup.ts.
You can update in current PR or create other PR for it as you refer.